ios/brave-ios/App/l10n/tools/push-strings-to-crowdin.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Push string changes to Transifex
+#
+# Error Codes:
+#   0 = Success
+#   1 = API Token must be set
+#   2 = Unauthorized access
+#   3 = Failed to export strings from Xcode project
+#   4 = Failed to cleanup strings
+#   5 = Failed to push string changes to Crowdin
+
+report_error()
+{
+  echo $2
+  echo $2 >>output.log 2>&1
+  cleanup
+  exit $1
+}
+
+cleanup()
+{
+  if [ -e transifex.log ] ; then
+    cat transifex.log >> output.log
+    echo >> output.log
+    rm transifex.log
+  fi
+
+  if [ -e ../../Client/en.xcloc ]; then
+    rm -R ../../Client/en.xcloc
+  fi
+
+  if [ -e en.xliff ] ; then
+    rm en.xliff
+  fi
+}
+
+if [ "${TOKEN}" = "" ] ; then
+  report_error 1 "TOKEN environment variable must be set to \"api\""
+fi
+
+cd $(dirname "$0")
+
+echo "Exporting strings from Xcode project..."
+(cd ../../ && xcodebuild -exportLocalizations SWIFT_EMIT_LOC_STRINGS=NO) >>output.log 2>&1
```

Comment:
If we only ever push up the `en.xliff` why dont we just only export english here? (`-exportLanguage en`)

Reply:
updated in[703406a](https://github.com/brave/brave-core/pull/27230/commits/703406ad7301f0b75556ab25732527ffcc0b75be)